.cargo/config.toml.offline

```diff
@@ -4,14 +4,9 @@ linker = "aarch64-linux-gnu-gcc"
 [source.crates-io]
 replace-with = "vendored-sources"
 
-[source."https://github.com/zcash/incrementalmerkletree.git"]
-git = "https://github.com/zcash/incrementalmerkletree.git"
-rev = "337f59179eda51261e9ddfc6b18e8fb84ea277c9"
-replace-with = "vendored-sources"
-
 [source."https://github.com/zcash/librustzcash.git"]
 git = "https://github.com/zcash/librustzcash.git"
-rev = "5a4a3e06dcd2cf5bdf79a7cd48709b58693c65f0"
+rev = "f69316e5bd940d5e4760c1ee0b0f3bef56d71534"
```

Comment:
This hash is from last week; it includes https://github.com/zcash/librustzcash/pull/1493 but not https://github.com/zcash/librustzcash/pull/1497, https://github.com/zcash/librustzcash/pull/1499, https://github.com/zcash/librustzcash/pull/1500, https://github.com/zcash/librustzcash/pull/1501, https://github.com/zcash/librustzcash/pull/1502, or https://github.com/zcash/librustzcash/pull/1503.

Reply:
It looks to me like we probably shouldn't have this in config.toml.offline at all? https://github.com/zcash/librustzcash/commit/16ba891726db9b8e39d825674b188ae5f60bed7d

Reply:
The bizarre thing is that f69316e5bd940d5e4760c1ee0b0f3bef56d71534 is a commit in this repository, not in `zcash/librustzcash`. I'm not sure what's going on here; I didn't manually update this at any point.

Reply:
No it is definitely in librustzcash: https://github.com/zcash/librustzcash/commit/f69316e5bd940d5e4760c1ee0b0f3bef56d71534